File: buildSrc/src/main/java/org/briarproject/briar/desktop/builddata/BuildDataPluginExtension.java
package org.briarproject.briar.desktop.builddata;

import org.gradle.api.tasks.Input;
import org.gradle.api.tasks.Optional;

public class BuildDataPluginExtension {

	@Input
	private String packageName;
	@Input
	@Optional
	private String className;

	public String getPackageName() {
		return packageName;
	}

	public void setPackageName(String packageName) {
		this.packageName = packageName;
	}

	public String getClassName() {
		return className;
	}

	public void setClassName(String className) {
		this.className = className;
	}

}

File: buildSrc/src/main/kotlin/org/briarproject/briar/desktop/builddata/BuildDataPluginExtension.kt
package org.briarproject.briar.desktop.builddata

import org.gradle.api.tasks.Input
import org.gradle.api.tasks.Optional

open class BuildDataPluginExtension {
    @Input
    var packageName: String? = null

    @Input
    @Optional
    var className: String? = null
}
